firestore-counter/functions/lib/controller.js

```diff
@@ -191,7 +191,7 @@ class ShardedCounterController {
                     yield t.get(this.controllerDocRef);
                 }
                 catch (err) {
-                    console.log("Failed to read controler doc " + this.controllerDocRef.path);
+                    console.log("Failed to read controller doc " + this.controllerDocRef.path);
```

Comment:
Should this be "Failed to read aggregator doc "

Reply:
Actually, this whole script and its contents use "controller" rather than "aggregator".  Should everything be updated to align with the other updates to the extension in this PR?  Or am I not understanding where the "controller" vs "aggregator" changes are applicable.

Reply:
Hmmm I'm thinking that we don't rename to `aggregator` since it may confuse existing users (we haven't change the functionality of it, we've just moved it from HTTP to PubSub). @rachelsaunders for context, we have to rename the function in order to change the trigger type. Can you think of a better name? `controller-ps`?

Reply:
Rachel and I discussed, and our best idea so far is changing it to `controllerFunc` so we are changing the name as little as possible and misleading people into thinking that the functionality has changed.

Reply:
This discussion is now out of date, see https://github.com/firebase/extensions/pull/49#issuecomment-559302743